claim/README.md

```diff
@@ -280,6 +280,8 @@ the claiming script parameters. For example, using the default claiming script:
 sudo netdata-claim.sh -token=TOKEN -rooms=ROOM1,ROOM2 -url=https://app.netdata.cloud -id=$(uuidgen)
 ```
 
+The agent *must be restarted* after this change.
```

Comment:
Sorry to be nitpicky, but our [Markdown linter](https://github.com/netdata/netdata/blob/master/.remarkrc.js) (however ignored it usually is), requires we use underscores for italics, not apostrophes.

```suggestion
The agent _must be restarted_ after this change.
```